src/AgGrid.fs

```diff
@@ -58,6 +58,12 @@ type IColumn = { getColId: unit -> string }
 [<Erase>]
 type IColumnDefProp<'row, 'value> = interface end
 
+type IRowHeightParameters<'row> = {
+    data: 'row option
+    RowNode: IRowNode<'row>
```

Comment:
```suggestion
    node: IRowNode<'row>
```

Based on [the docs](https://www.ag-grid.com/react-data-grid/row-height/#getrowheight-callback), shouldn't this be called `node`? I think the name needs to line up for the code generated by Fable to work properly. Even if it's not required, I think that it's best to stick as closely as possible to the terminology that AG Grid uses, so that developers don't have to translate between this F# library's code and what's in the AG Grid docs.

I also notice that RowHeightParams includes a context field. Might be worth adding, even if you don't need it, so that we don't need to add it separately in future if someone else needs it. Not necessary if that's too complex for now though.